### PR TITLE
OHRM5X-933: Add CSRF token to login page

### PR DESCRIPTION
**Pull request by RajithaKumara**
5 x csrf drawio

### CODE REVIEW

File: symfony/client/src/orangehrmAuthenticationPlugin/pages/Login.vue [hunk 1]
```diff
@@ -119,6 +126,12 @@ export default {
       this.$refs.loginForm.$el.submit();
     },
   },
+
+  beforeMount() {
+    setTimeout(() => {
+      reloadPage();
+    }, 1200000); // 20 * 60 * 1000 (20 minutes);
+  },
```
Comment (RajithaKumara) [inline symfony/client/src/orangehrmAuthenticationPlugin/pages/Login.vue:134]: Added this to address;
https://github.com/orangehrm/orangehrm/commit/a9da037055e5399edbbfadfdaa548b63d24937c6#diff-f51a7389083b23ea24d1c5dac16a022305099f4911532b98ad27da3df252a4ed

File: symfony/plugins/orangehrmCorePlugin/test/Dao/AccessFlowStateMachineDaoTest.php [hunk 2]
```diff
@@ -261,15 +261,11 @@ public function testHandleUserRoleRename(): void
 
     protected function getWorkflowItemIdsForRole($flow, $role)
     {
-        $conn = $this->getEntityManager()->getConnection();
+        $conn = $this->getEntityManager()->getConnection()->getWrappedConnection();
```
Comment (lgtkaushalya) [inline symfony/plugins/orangehrmCorePlugin/test/Dao/AccessFlowStateMachineDaoTest.php:264]: Are these changes related to the change?

Comment (RajithaKumara) [reply symfony/plugins/orangehrmCorePlugin/test/Dao/AccessFlowStateMachineDaoTest.php:264]: No. Upgrade doctrine 2.8 to 2.10, then had to refactor code.
This is the CSRF token related commit: https://github.com/orangehrm/orangehrm/pull/1002/commits/1b215cbd2f44b8d1be99102294e45c6bb41c12cd

**Review by samanthajayasinghe — APPROVED** [review]
